odl/solvers/functional/default_functionals.py

```diff
@@ -637,6 +645,14 @@ class L2NormSquared(Functional):
 
     .. math::
         \| x \|_2^2 = \\int_\Omega |x(t)|^2 dt.
+
+    The `proximal` factory allows using vector-valued stepsizes:
+
+    >>> space = odl.rn(3)
+    >>> f = odl.solvers.L2NormSquared(space)
+    >>> x = space.one()
+    >>> f.proximal([0.5, 1.0, 1.5])(x)
+    rn(3).element([ 0.5       ,  0.33333333,  0.25      ])
```

Comment:
Please use numbers that can be computed and represented exactly, e.g. `[0.5, 1.5, 2.0]` I guess. Printing approximate numbers may depend on Python version and Numpy version and ruin the doctest.